labels/mapping.go

```diff
@@ -21,11 +47,11 @@ var core = []Label{
 
 var plugin = []Label{
 	{"Bug", "Something isn't working", "d73a4a"},
-	{"Duplicate", "This issue or pull request already exists", "cfd3d7"},
+	// {"Duplicate", "This issue or pull request already exists", "cfd3d7"},
```

Comment:
I don't really see us using resolution statuses for plugins, closing the issue with a comment should be enough?

Reply:
To me this helps making clear why an issue was closed. It's also a very common label on GH. 3/5